common/nginx/etc/nginx.conf

```diff
@@ -30,6 +30,13 @@ http {
                         proxy_redirect off;
                 }
 
+                location /soroban/rpc/admin {
```

Comment:
I think we should make this a separate port. While this image is used for testing, it's also possible to deploy it to a public URL for testing, and it should be reasonable to expose admin information in these settings on another port.

A separate port is common practice?

Reply:
That would make this easier to implement. You don't need nginx. Just expose it on another fixed point, and folks can choose to map it in docker land, like we do the variety of other ports the instance exposes for admin usage.

Reply:
sounds good, reverted to that approach - https://github.com/stellar/quickstart/pull/463/commits/c8678f41c5a3b3e59138499cfba03221a4117744